lib/oci_utils/impl/network_helpers.py

```diff
@@ -35,6 +36,15 @@
 _NM_CONF_DIR = "/etc/NetworkManager/conf.d/"
 _logger = logging.getLogger('oci-utils.net-helper')
 
+def is_network_namespace_exists(name):
+    """
+    Checks that a namespace exist or not
+    parameter:
+      name : namespace name as str
+    Returns:
+       True if exists False otherwise
+    """
+    return os.path.exists('/var/run/netns/%s'%name)
```

Comment:
minor thing, doc is not numpy ...

Reply:
Hi,
not sure to get what's wrong . 
any example ?

Reply:
> minor thing, doc is not numpy ...

Hi,
not sure to get what's wrong . 
any example ?

Reply:
just formatting, .. :) 
    """
    Checks that a namespace exist or not.

    Parameters
    ---------------
    name : namespace name as str.

    Returns
    -----------
    True if exists False otherwise.
    """